app/src/main/java/com/example/internetshop/presentation/activity/fragments/AuthenticationFragment.kt

```diff
@@ -22,7 +22,7 @@ class AuthenticationFragment : BaseFragment() {
         component.inject(this)
     }
 
-    override fun getTitle(): String = context?.resources?.getString(R.string.login)?:""
+    override fun getTitle(): String = context?.resources?.getString(R.string.login) ?: ""
```

Comment:
Redundant change. Please don't align the whole file if the file is not new. You should to align only those lines which were changed